web/src/app/pages/create-survey/create-survey.component.ts

```diff
@@ -52,18 +52,38 @@ export class CreateSurveyComponent implements OnInit {
       this.currentSurveyId = surveyId;
     });
     this.surveyService.getActiveSurvey$().subscribe(survey => {
+      if (this.isSetupFinished(survey)) {
+        this.navigationService.navigateToEditSurvey(survey.id);
+        return;
+      }
       this.currentSurvey = survey;
       this.setupPhase = this.getSetupPhase(survey);
     });
   }
 
+  private isSetupFinished(survey: Survey): boolean {
```

Comment:
What about LOIs?

Reply:
We don't need to since task is the next step of LOI, so I wanted to make it simple and only check tasks. I added a comment to explain why.